components/SCodeBlock/index.tsx

```diff
@@ -93,10 +95,10 @@ export const SCodeBlock = ({ path }: SCodeBlockProps) => {
           "Content-Type": "application/json",
         },
         body: JSON.stringify({
-          language: languages[idx] ? languages[idx].pistonName : "python", // specify the language
-          files: [{"name": "sol." + (languages[idx] ? languages[idx].extension : "py"), "content": codes[idx]}], // specify the code to execute
+          language: languages[idx]!.pistonName, // specify the language
+          files: [{"name": "sol." + languages[idx]!.extension, "content": codes[idx]}], // specify the code to execute
           stdin: testCase, // specify the input
-          version: languages[idx] ? languages[idx].version : "3.10",
+          version: languages[idx]!,
```

Comment:
I think this is meant to be `languages[idx]!.version`.

Reply:
yep also noticed that, moment of pure idiocy